terraform/rob-cos/variables.tf

```diff
@@ -0,0 +1,53 @@
+variable "model" {
+  description = "Name of the model to deploy to (must be a K8s model)"
+  type        = string
+  nullable    = false
+}
+
+variable "blackbox_exporter" {
+  type = object({
+    channel  = optional(string, "1/stable")
+    revision = optional(number)
+  })
+  default     = {}
+  description = <<-EOT
+  The blackbox-exporter variables.
+  Please refer to the module for more information.
+  EOT
+}
+
+variable "cos_lite" {
+  type = object({
+    channel = optional(string, "1/stable")
+    use_tls = optional(bool, false)
+  })
+  default     = {}
+  description = <<-EOT
+  The cos-lite variables.
+  Please refer to the module for more information.
+  EOT
+}
+
+variable "cos_registration_server" {
+  type = object({
+    channel  = optional(string, "latest/edge")
```

Comment:
I think we should put another channel here (beta for example) since merging anything to the cos-registration-server-k8s could break something.
We can promote the actual edge to beta.